<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrearCliente</class>
 <widget class="QMainWindow" name="CrearCliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Crear Cliente</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="FrameFormulario">
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>551</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    border: 2px solid #e4acd0; /* Borde */
    border-radius: 5px;
    padding: 5px;
    background-color: #f5eef2; /* Fondo */ 
    font-family: Arial, sans-serif;
    font-size: 14px;
}
QLineEdit:hover {
	border: 2px solid rgb(255, 255, 255);
}
QLineEdit:focus {
    border: 2px solid #dc84bc; /* Borde al enfocar */
}

QLabel {
    background-color: transparent; /* Fondo transparente */
    color: #333333; /* Color del texto */
    font-family: Arial, sans-serif; /* Fuente del texto */
    font-size: 16px; /* Tamaño de la letra */
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="IngresoCedula">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>380</width>
         <height>32</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Cedula</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="IngresoNombre">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>240</y>
         <width>380</width>
         <height>32</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Nombre</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="IngresoTelefono">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>400</y>
         <width>380</width>
         <height>32</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Teléfono</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Ingresar Cedula</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Ingresar Nombre</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>370</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Ingresar Teléfono</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="FrameBotones">
      <property name="maximumSize">
       <size>
        <width>113</width>
        <height>115</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: 2px solid #e4acd0;
    border-radius: 5px;
    padding: 8px 16px;
    background-color: #dc95c4; /* Fondo */
    color: #ffffff; /* Color del texto */
    font-family: Arial, sans-serif;
    font-size: 16px;
}

QPushButton:hover {
    background-color: #dc84bc; /* Fondo al pasar el cursor */
}

QPushButton:pressed {
    background-color: #d484b4; /* Fondo al presionar */
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="BotonAtrasCC">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>111</width>
         <height>51</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>111</width>
         <height>51</height>
        </size>
       </property>
       <property name="text">
        <string>Atras</string>
       </property>
      </widget>
      <widget class="QPushButton" name="BotonGuardar">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>62</y>
         <width>111</width>
         <height>51</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>111</width>
         <height>51</height>
        </size>
       </property>
       <property name="text">
        <string>Guardar</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
